### PR TITLE
Add versioning metadata for #1123

### PR DESCRIPTION
**Pull request by dampee**
I would like to add versioning, but this does not seem to work on INDEX files

### CODE REVIEW

File: Extending/ContentApps/index-v8.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+versionFrom:8.0.0
```
Comment (jmayntzhusen) [inline Extending/ContentApps/index-v8.md]:
```suggestion
versionFrom: 8.0.0
```

File: Extending/ContentApps/index.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+versionTo:7.99.0
```
Comment (jmayntzhusen) [inline Extending/ContentApps/index.md]:
```suggestion
versionFrom: 7.0.0
```

File: Extending/index-v8.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+---
+versionFrom:8.0.0
```
Comment (jmayntzhusen) [inline Extending/index-v8.md]:
```suggestion
versionFrom: 8.0.0
```

File: Extending/index.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+---
+versionFrom:7.0.0
```
Comment (jmayntzhusen) [inline Extending/index.md]:
```suggestion
versionFrom: 7.0.0
```

**Comment by dampee** [comment]
I'll do it as soon as i am in the office

**Pull request merged**